### ISSUE TITLE
Support for SDL2 in Cmake and LvDisplay improvements and Migration to v9

### ISSUE THREAD

**Issue by spi3ex** [lvgl/lv_binding_cpp#16]
Updates are here: https://github.com/spi3ex/lv_binding_cpp/tree/staging
If anyone wants to take it forward, Please take this
I'm still working on it, as of now 60% Complete.

Planned work:
- Improve Cmake for linking and source
- Improve for LvDisplay and Draw Buffers
- Migrate to v9

**Comment by kisvegabor** [comment]
Hi,

Thank you for the update! I couldn't find a script in the repo to generate binding. Is it written manually now?

**Comment by spi3ex** [comment]
> Hi,
> 
> Thank you for the update! I couldn't find a script in the repo to generate binding. Is it written manually now?

Yes, it is manually Written, as we need to port all API's to V9. 
As With Cpp we're using wrapper functions to call CPP Objects to lvgl C API
Major issues are with **lv_drivers**/ instead of lvgl/src/drivers/

**Comment by kisvegabor** [comment]
How complicated would it be to auto generate the wrappers?

> Major issues are with lv_drivers/ instead of lvgl/src/drivers/

Do you mean it's complicated to move from `lv_drivers` to `lvgl/src/drivers`?
